### PR TITLE
fix: ApolloClient initialisation to use cross-fetch in SubgraphClient

### PR DESCRIPTION
**Pull request by kanthgithub**
# Description
During startup when SubgraphClient is initialised which internally creates ApolloClient it throws error
More details are added to the Linked Issue

## Issue
https://github.com/bcnmy/userop-debugger-backend/issues/31

## Change Details
1. add dependency for `cross-fetch`
2. update ApolloClient initialisation which uses `fetch`
```js
this.client = new ApolloClient({
            link: new HttpLink({ uri: config.uri, fetch }),
            cache: new InMemoryCache()
            });
```

## Post fix verification

- Startup is successful

```sh
➜  userop-debugger-backend git:(fix/apollo-client-init-issue) yarn start:dev      
yarn run v1.22.19
$ nodemon
[nodemon] 3.0.2
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): src/**/*
[nodemon] watching extensions: ts
[nodemon] starting `yarn start`
$ node --inspect=0.0.0.0:5858 -r ts-node/register ./src/start-dev.ts
Debugger listening on ws://0.0.0.0:5858/9c4ba630-cc5c-421a-87e2-bb00d31dd47e
For help, see: https://nodejs.org/en/docs/inspector
Server started on port  8000
```

### CODE REVIEW

File: src/service/subgraph/index.ts [hunk 1]
```diff
@@ -14,9 +15,9 @@ export class SubGraphClient {
         }
         
         this.client = new ApolloClient({
-            uri: config.uri,
+            link: new HttpLink({ uri: config.uri, fetch }),
             cache: new InMemoryCache()
-        });
+            });
```
Comment (tomarsachin2271) [inline src/service/subgraph/index.ts:20]: Please check the indentation here.

Comment (kanthgithub) [reply src/service/subgraph/index.ts]: * updated the indentation

shall i add prettier dependency, update script in package.json for prettier formatting?
(in a separate PR) ?

@tomarsachin2271

Comment (tomarsachin2271) [reply src/service/subgraph/index.ts]: Yeah i think it's better to add prettier dependency

Comment (tomarsachin2271) [reply src/service/subgraph/index.ts]: Merging this PR.. let's add another PR for prettier

Comment (kanthgithub) [reply src/service/subgraph/index.ts]: i will make it on separate PR after approval and merge for this PR

**Review by tomarsachin2271 — APPROVED** [review]
Looks good 🚀 LGTM

**Pull request merged**